ballerina/http_connection.bal

```diff
@@ -141,8 +144,8 @@ public isolated client class Caller {
                 InternalServerError errResponse = {};
                 response = createStatusCodeResponse(errResponse);
             } else {
-                Accepted AcceptedResponse = {};
-                response = createStatusCodeResponse(AcceptedResponse);
+                Accepted constructedResponse = {};
```

Comment:
Can we roll-back this change and have the variable name as `AcceptedResponse`?
I think that is more meaningful